### ISSUE TITLE
Inviting user causes Umbaco to hang

### ISSUE THREAD

**Issue by ArnsteinBerg** [umbraco/Umbraco-CMS#10871]
### Which *exact* Umbraco version are you using? For example: 8.13.1 - don't just write v8

8.15.1

### Bug summary

When inviting a user in Umbraco the system hangs, using 100% CPU until the web server is restarted. The user is created, but the invite email is not sent. Same thing happens if I later select "Resend invitation".

### Specifics

Tested both on Azure App service and IIS on my local PC with same result. No errors in log, just the CPU spike. Standard SMTP mail configuration. The whole server setup was working up until Umbraco v8.8.

### Steps to reproduce

Invite a user, then check CPU usage.

### Expected result / actual result

CPU usage should remain normal after invite, and the user should receive invite.
Actual result is no email sent (nothing in mail server log), and CPU spike at 100% and not recovering.

**Comment by nathanwoulfe** [comment]
Hery @ArnsteinBerg I've just checked this on the latest from v8/contrib, it appears to be working fine - email sent, no CPU spike:

![image](https://user-images.githubusercontent.com/3248070/129660114-e2994edc-0582-4578-887c-87c9ed86000e.png)

My SMTP config looks like the below, nothing special:

```xml
<smtp deliveryMethod="Network" from="test@example.com">
  <network host="127.0.0.1" port="25"/>
</smtp>
```

Is there anything non-standard/custom in your site that could be causing this?

**Comment by nul800sebastiaan** [comment]
Thanks, this is fixed in today's 8.15.2 release: https://github.com/umbraco/Umbraco-CMS/issues/10746

**Comment by ArnsteinBerg** [comment]
Great, then it wasn't just my environment. I will try the fix then. Thank you very much!

During the testing this morning I found another issue though. It seems that you handle missing locales differently now, and instead of displaying the English version as fallback, you now just display the dictionary key. I have some users that use Norwegian (nb), and the emails are not translated. Earlier we got the English email, but now the content of the password reset email is just "[resetPasswordEmailCopyFormat]". It also makes the backoffice look a bit ugly (see picture), but the missing email content is definitely more of a problem.
![Umbraco nb](https://user-images.githubusercontent.com/3405706/129683617-377a7209-58b3-470e-8bb6-7db5e6896acb.PNG)
 Should I post this as a separate bug, or are you familiar with it?
